R/modules.R

```diff
@@ -217,7 +217,8 @@ module <- function(label = "module",
       "\n - `...` ui_args elements will be passed to the module argument of the same name or to the `...`"
     )
   }
-  if (any(c("data", "datasets") %in% ui_formals)) {
+
+  if (any("datasets" %in% ui_formals)) {
```

Comment:
Please revert. We don't support any data in the ui functions - data related content can be created only via server functions. Thanks to this UI "skeleton" can be rendered on init without a need to wait on data.


```suggestion
  if (any(c("data", "datasets") %in% ui_formals)) {
```